### PR TITLE
Return IDs of deleted workflow runs in unload endpoint

### PR DESCRIPTION
**Pull request by callunity**
GP-2862

### CODE REVIEW

**Comment by callunity** [comment]
Closing as I opened this a bit prematurely.

**Review by mlaszloffy — APPROVED** [review]

**Comment by callunity** [comment]
(Reverted to draft as I discovered that unload wasn't operating recursively bug)

**Comment by callunity** [comment]
Reopening as I've developed a fix for the recursive unload/copy-out, and it's based off this code

File: vidarr-server/src/test/java/ca/on/oicr/gsi/vidarr/server/MainIntegrationTest.java [hunk 1]
```diff
@@ -1071,8 +1070,8 @@ public void whenWorkflowRunsAreUnloaded_thenTheyCanBeLoaded() throws IOException
             .statusCode(200)
             .and()
             .extract()
-            .asInputStream();
-    var unload2FileName = MAPPER.readTree(unload2FileQuotedName).toString().replaceAll("\"", "");
+            .jsonPath();
+    var unload2FileName = res.get("filename").toString();
```
Comment (avarsava) [inline vidarr-server/src/test/java/ca/on/oicr/gsi/vidarr/server/MainIntegrationTest.java:1074]: should unload2FileName come from getting filename from res2?

Comment (callunity) [reply vidarr-server/src/test/java/ca/on/oicr/gsi/vidarr/server/MainIntegrationTest.java]: Yes, thanks for catching that! I've updated the test.

**Pull request merged**